core/src/main/java/org/apache/spark/api/java/function/package.scala

```diff
@@ -22,4 +22,4 @@ package org.apache.spark.api.java
  * these interfaces to pass functions to various Java API methods for Spark. Please visit Spark's
  * Java programming guide for more details.
  */
-package object function 
+package object function
```

Comment:
Could you take a look my PR again? Or, 

https://git-wip-us.apache.org/repos/asf?p=spark.git;a=blob;f=core/src/main/java/org/apache/spark/api/java/function/package.scala;h=0f9bac716416264aeba175b90c0b32570bc6dd81;hb=HEAD